eodatasets3/properties.py

```diff
@@ -313,6 +313,7 @@ def parsed_sentinel_datastrip_id(tile_id) -> tuple[str, dict]:
 _STAC_MISC_PROPS = {
     "providers": None,  # https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md#provider,
     # Projection extension
+    "proj:code": str,
```

Comment:
I don't know anything about eo-datasets and how the code is supposed to work, I'm just guessing that `proj:code` should be alongside wherever `proj:epsg` was before.